explore-analyze/_snippets/global-filters.md

```diff
@@ -0,0 +1,24 @@
+You can interact with filter pills to edit them, temporarily alter their behavior without having to delete them, or make them available in other contexts. The following options are available when selecting a filter pill:
+
+**Pin across all apps** and **Unpin**
```

Comment:
Another interesting behavior is that Pinned filters are not saved. For example, if you pin a filter on a dashboard and save the dashboard and then open the dashboard in a new tab, the pinned filter is no longer present.  Not sure if this is worth mentioning.

Reply:
I think it is! I will add something